packages/js/components/src/experimental-tree-control/index.ts

```diff
@@ -0,0 +1,4 @@
+export * from './tree';
```

Comment:
Minor and may not be necessary for this PR, but should we also export this under the `components/src/index.ts`?